ENH raise error for invalid 'install_cmd' (#714)
benchopt/benchopt: doc/benchmark_workflow/install_benchmark.rst

.. _install_benchmark:

Install a benchmark
======================

In order to make it easy to run a new benchmark, benchopt provides an interface
to specify and install requirements for the various component of the benchmarks.

> - The minimal requirements that are necessary to run the benchmark are
>   specified in ``objective.py``. They can be installed using the command
>   ``benchopt install --minimal``.
> 

Reviewer: ```suggestion
- By default, calling ``benchopt install .`` will install the requirements for the
   benchmark, including all solvers and datasets.

- The minimum requirements to run the benchmark are specified in
  ``objective.py``. They can be installed using the command
  ``benchopt install --minimal``.

```

- The requirements that are specific to each ``Dataset/Solver`` can be
  specified in each class, and they can be installed individually by selecting
  the proper component using ``benchopt install -d dataset1 -s solver1``.

.. _specify_requirements:

> Specify requirements
> -----------------------

Reviewer: ```suggestion
Specifying requirements
-----------------------
```

To specify how dependencies should be installed, you can use the ``install_cmd`` class attribute.
This attribute accepts two possible values:

1. ``conda`` (default): Dependencies will be installed using Conda. In this case, you should 

>   specify the required dependencies in the ``requirements`` class attribute.

Reviewer: ```suggestion
  specify the required dependencies in the ``requirements`` class attribute. Note that
  dependencies to install with ``pip`` are also specified with this option.
```
